src/main/java/ru/practicum/shareit/exception/handle/ErrorResponce.java

```diff
@@ -0,0 +1,15 @@
+package ru.practicum.shareit.exception.handle;
+
+import lombok.Getter;
+
+public class ErrorResponce {
```

Comment:
Здесь опечатка, не критично
```suggestion
public class ErrorResponse {
```